src/Migrations/M014RemoveDropdownNavigationMenusOption.php

```diff
@@ -6,7 +6,6 @@
 
 use P4\MasterTheme\MigrationRecord;
 use P4\MasterTheme\MigrationScript;
-use P4\MasterTheme\Settings\InformationArchitecture;
```

Comment:
I adjysted these two migrations, because if an instance hasn't run them already they will crash since `P4\MasterTheme\Settings\InformationArchitecture` doesn't existin anymore.